packages/map-template/src/components/Map/Map.jsx

```diff
@@ -205,11 +205,20 @@ function Map({ onLocationClick, onVenueChangedOnMap }) {
      * @param {object} positionControl - MapsIndoors PositionControl instance.
      */
     const onPositionControlCreated = positionControl => {
-        positionControl.on('position_received', positionInfo => {
-            if (positionInfo.accurate === true) {
-                setUserPosition(positionInfo.position);
-            }
-        });
+        if (document.querySelector('mi-my-position')) {
```

Comment:
Could we just as well use `positionControl.nodeName` to check for the position control type? I feel that is more clean than querying the DOM.